includes/JWTHandler.php

```diff
@@ -27,17 +27,20 @@ public function __construct(
     public function preprocessRawJWTData(
         string $rawJWTData
     ): string {
-        if (!str_starts_with($rawJWTData, 'Bearer:')) {
+        // Checks if the $rawJWTData string starts with the literal strings "Bearer:" or "Bearer ".
+        // If it doesn't start with either, it logs an error message and returns an empty string.
+        if (strpos($rawJWTData, 'Bearer:') !== 0 && strpos($rawJWTData, 'Bearer ') !== 0) {
             $this->logger->debug("Invalid JWT auth, doesn't start with Bearer");
             return '';
         }
-
-        $rawJWTData = substr($rawJWTData, strlen('Bearer:'));
+        // Extract the JWT substring after the first 7 characters, which removes the "Bearer:" or "Bearer " prefix.
+        // Remove any white space characters from the $rawJWTData string and output as $cleanJWTData.
+        $rawJWTData = str_replace(array('Bearer:', 'Bearer '), '', $rawJWTData);
```

Comment:
Nit: Would it be possible to use the new array syntax? `['Bearer:', 'Bearer ']` instead of `array(['Bearer ', 'Bearer:'])`.

Reply:
Incorporated